sdl_android/src/main/java/com/smartdevicelink/api/SdlManager.java

```diff
@@ -380,6 +404,20 @@ public void sendRPCs(List<? extends RPCMessage> rpcs, final OnMultipleRequestLis
 
 	// LIFECYCLE / OTHER
 
+	// STARTUP
+	public void start(CompletionListener listener){
+		initListener = listener;
+		if(initialized){
+			listener.onComplete(true);
+		}
+	}
+
+	@Override
+	public void onProxyConnected() {
```

Comment:
After further review, I believe the SdlManager class should create an anonymous class of the Lifecycle Listener interface instead of implement. These methods should not be exposed to the developer.